x/feeshare/README.md

```diff
@@ -0,0 +1,6 @@
+# fee-share
```

Comment:
Would be nice to have a little more documentation for a custom module such as this. :heart:

Reply:
Resolved -> https://github.com/CosmosContracts/juno/pull/385/commits/91d60e422c49c3567535df0460a3cc3f9e0a387f